manifests/d/denoland/deno/1.25.3/denoland.deno.locale.en-US.yaml

```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.6 $debug=NVS1.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/manifests/v1.4.0/manifest.defaultLocale.1.4.0.json
+
+PackageIdentifier: denoland.deno
+PackageVersion: 1.25.3
+PackageLocale: en-US
+Publisher: denoland
+PublisherUrl: https://github.com/denoland
+PublisherSupportUrl: https://github.com/denoland/deno/issues
+PrivacyUrl: https://deno.com/deploy/docs/privacy-policy
+# Author:
+PackageName: deno
+PackageUrl: https://deno.land/
+License: MIT
+LicenseUrl: https://github.com/denoland/deno/blob/main/LICENSE.md
+# Copyright:
+# CopyrightUrl:
+ShortDescription: A modern runtime for JavaScript and TypeScript
+Description: Deno is a simple, modern and secure runtime for JavaScript, TypeScript, and WebAssembly that uses V8 and is built in Rust.
+# Moniker:
+Tags:
+- javascript
+- runtime
+- typescript
+# Agreements:
+# ReleaseNotes:
+ReleaseNotesUrl: https://github.com/denoland/deno/releases/tag/v1.25.3
+PurchaseUrl: https://deno.com/deploy/pricing
```

Comment:
I think the pricing URL should be removed. Deno Deploy is a completely separate product.